doc/ipython-notebooks/clustering/KMeans.ipynb

```diff
@@ -1,1078 +1,1041 @@
 {
+ "cells": [
```

Comment:
as you can see, this diff here is everything. Can you send a separate PR that just stores the nb in the new format and then update this one here to just have your changes? thx